Gulpfile.js

```diff
@@ -38,14 +41,25 @@ function compile(advanced = false) {
         .pipe(sourcemaps.init())
         .pipe(closureCompiler(options))
         .pipe(sourcemaps.write('/'))
-        .pipe(gulp.dest('dist'));
+        .pipe(gulp.dest('dist/js'));
+}
+
+function cssMinify() {
+    return gulp.src('src/components/**/*.css')
+        .pipe(concatCss('css/erste.css'))
```

Comment:
> You should be able to use erste.min.css here and remove the rename.

That makes sense 👍 

I would say let's use `erste.min.js`. When I did that change which places I should change?